set.mm

```diff
@@ -17242,26 +17242,19 @@ it were an axiom (as it is in the case of Tarski).
       ( weq wex wa ax6evr equtr ancrd eximdv mpi ) ABDZBCDZCEACDZMFZCECBGLMOCLM
       NABCHIJK $.
 
-    $( A specialized version of ~ equvel with distinct variable restrictions
-       and fewer axiom usage.  (Contributed by Andrew Salmon, 2-Jun-2011.)
+    $( A biconditional form of ~ equvel with DV conditions and proved from
```

Comment:
the dv shorthand is lowercase elsewhere.  Is it explained somewhere, so people new to Metamath know how to interpret it?

Reply:
> the dv shorthand is lowercase elsewhere. Is it explained somewhere, so people new to Metamath know how to interpret it?

That's partially true: "dv condition" appears 174 times, "DV condition" 30 times. I would propose to unify it to "DV condition".

Our conventions contain the list item:

<li>No two variables occurring in the LHS may share a
       disjoint variable (DV) condition.
       </li>

(This should read "distinct variable (DV) condition", shouldn't it? (see https://us.metamath.org/mpeuni/mmset.html#distinct). But we have (at least) 11 "disjoint variable condition" in set.mm (and 106 "distinct variable condition").

Reply:
Disjoint is my favorite here, because of $d x ph $.  What about ~ DV and a fake theorem page explaining it?

Reply:
The error message of the metamath program also says "disjoint variable requirement". I think there was already a discussion about these terms, but I do not know where...

Reply:
I think this is a non-blocking issue

Reply:
OK, I will create an Github issue for the alignment of the terms disjoint/distinct variables.